README.md

```diff
@@ -147,11 +147,11 @@ Depending on how ORT was installed, it can be run in the following ways:
 
   Note that in this case the working directory used by ORT is that of the `cli` project, not the directory `gradlew` is located in (see https://github.com/gradle/gradle/issues/6074).
 
-# Want to Help or have Questions?
+# Contributing
 
 All contributions are welcome.
-If you are interested in contributing, please read our [contributing guide](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md).
-To get quick answers to any of your questions, we recommend you [join our Slack community][2].
+If you are interested in contributing code, please read our [contributing guide](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md).
```

Comment:
Why do you think should the `CONTRIBUTING.md` be only about code contributions, but not about any other contributions? (If we'd a further info about non code contributions, would that file be the place to go?)

Apart from that, the last section of that file is about project governance, which is not strictly just about code.

All in all, I'm not sure this aspect of the change is fine.

Reply:
> Why do you think should the `CONTRIBUTING.md` be only about code contributions

I'm not saying it should be that way going forward, but current it is, i.e. `CONTRIBUTING.md` talks bout Git history and coding conventions, and thus *currently* is only relevant for code contributions.

> If we'd a further info about non code contributions, would that file be the place to go?

Basically yes, but to me it depends on concretely what kind of non-code contributions. E.g. guidelines for documentation contributions could be covered there, but for bug report contributions I'd prefer to canonically refer to the issue workflow / template.

> Apart from that, the last section of that file is about project governance, which is not strictly just about code.

Indeed. That section should probably be combined with the *License* section.

Reply:
> Apart from that, the last section of that file is about project governance, which is not strictly just about code.

See https://github.com/oss-review-toolkit/.github/pull/4.